MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ListView.xaml

```diff
@@ -88,7 +88,7 @@
             <Setter.Value>
                 <ControlTemplate TargetType="GridViewColumnHeader">
                     <DockPanel>
-                        <Thumb x:Name="PART_HeaderGripper"
+                        <wpf:GridViewColumnThumb x:Name="PART_HeaderGripper"
```

Comment:
Rather than changing the `MaterialDesignGridColumnHeaderGripper` template, lets simply do an explicit style on this element and base it on `MaterialDesignGridColumnHeaderGripper` and set the template here. This way we won't accidently change any other thumbs that may not be expecting this behavior.

Reply:
I created a new style named `MaterialDesignGridViewColumnHeaderGripper`. The GridView Gripper does not use ```<Setter Property="Cursor" Value="SizeWE"/>``` so I opted to set the base as `MaterialDesignThumb`